pkg/dispatcher/config.go

```diff
@@ -210,6 +211,54 @@ func LoadConfig(configPath string) (*Config, error) {
 	return config, nil
 }
 
+func (config *Config) JobsWithMultiOccurrence() []string {
+	records := map[string]int{}
+	for _, group := range config.Groups {
+		for _, job := range group.Jobs {
+			records[job] = records[job] + 1
```

Comment:
golang has `++`, why not use it?

Reply:
I cannot see `Outdated`, although i have updated the line.
/shrug

Reply:
I see you updated the occurrence below (L247) but not here

Reply:
I checked out the code form GH to be sure it does not lie ;)